### PR TITLE
VIX-3554 Restoring Elexol Ether IO Controller

### PR DESCRIPTION
**Pull request by johncbaur**
VIX-3554 Restoring Elexol Ether IO Controller

### CODE REVIEW

File: Vixen.sln [hunk 1]
```diff
@@ -2313,6 +2315,18 @@ Global
 		{82D80094-F012-4248-ABA7-D55FB22807CD}.Release|x64.Build.0 = Release|x64
 		{82D80094-F012-4248-ABA7-D55FB22807CD}.Release|x86.ActiveCfg = Release|x86
 		{82D80094-F012-4248-ABA7-D55FB22807CD}.Release|x86.Build.0 = Release|x86
+		{F246924F-3A0C-4E70-9969-C20FED06A0E3}.Debug|x64.ActiveCfg = Debug|x64
+		{F246924F-3A0C-4E70-9969-C20FED06A0E3}.Debug|x64.Build.0 = Debug|x64
+		{F246924F-3A0C-4E70-9969-C20FED06A0E3}.Debug|x86.ActiveCfg = Debug|x86
+		{F246924F-3A0C-4E70-9969-C20FED06A0E3}.Debug|x86.Build.0 = Debug|x86
+		{F246924F-3A0C-4E70-9969-C20FED06A0E3}.Deploy|x64.ActiveCfg = Release|x64
+		{F246924F-3A0C-4E70-9969-C20FED06A0E3}.Deploy|x64.Build.0 = Release|x64
+		{F246924F-3A0C-4E70-9969-C20FED06A0E3}.Deploy|x86.ActiveCfg = Debug|x86
```
Comment (jeffu231) [inline Vixen.sln:2324]: All the deploy targets should be Release. Looks like you missed x86 here even though we really don't use it.

File: Vixen.sln [hunk 2]
```diff
@@ -2305,8 +2305,8 @@ Global
 		{82D80094-F012-4248-ABA7-D55FB22807CD}.Debug|x64.Build.0 = Debug|x64
 		{82D80094-F012-4248-ABA7-D55FB22807CD}.Debug|x86.ActiveCfg = Debug|x86
 		{82D80094-F012-4248-ABA7-D55FB22807CD}.Debug|x86.Build.0 = Debug|x86
-		{82D80094-F012-4248-ABA7-D55FB22807CD}.Deploy|x64.ActiveCfg = Debug|x64
-		{82D80094-F012-4248-ABA7-D55FB22807CD}.Deploy|x64.Build.0 = Debug|x64
+		{82D80094-F012-4248-ABA7-D55FB22807CD}.Deploy|x64.ActiveCfg = Release|x64
+		{82D80094-F012-4248-ABA7-D55FB22807CD}.Deploy|x64.Build.0 = Release|x64
 		{82D80094-F012-4248-ABA7-D55FB22807CD}.Deploy|x86.ActiveCfg = Debug|x86
```
Comment (jeffu231) [inline Vixen.sln:2312]: I missed the debug ones on this one when I merged it also. Might as well fix it up as well.

**Review by jeffu231 — CHANGES_REQUESTED** [review]

File: Vixen.sln (hunk 2 above)
Comment (johncbaur) [reply Vixen.sln]: Edited file to specify release.

File: Vixen.sln (hunk 1 above)
Comment (johncbaur) [reply Vixen.sln]: Edited file to specify release.

Comment (johncbaur) [reply Vixen.sln]: I attempted to make the change in Visual Studio but it was splitting Any CPU into x86 and x64 which made the difference count much greater so I did it manually.

**Review by jeffu231 — APPROVED** [review]

**Pull request merged**